peachjam/js/peachjam.ts

```diff
@@ -137,6 +138,32 @@ class PeachJam {
     });
   }
 
+  setupPopovers () {
+    document.querySelectorAll('[data-bs-toggle="help-popover"]').forEach((el) => {
+      // @ts-ignore
+      const helpPopover = new window.bootstrap.Popover(el, {
+        html: true,
+        content: `
+        ${el.getAttribute('data-bs-content')}
+         <div><a href="${el.getAttribute('href')?.split('#')[1] || '#'}" target='_blank' rel='noopener noreferrer'>Learn more</a></div>
+        `,
+        container: 'body'
+      });
+      // keep popover open when clicking inside it but close when clicking outside it
+      el.addEventListener('inserted.bs.popover', (e) => {
```

Comment:
I don't think we need this -- if the user clicks the link in the popover, the docs open in a new tab and the popover can close? Or am I confused?

Reply:
@longhotsummer When the popover trigger is set to `click`, the popover can only be closed by clicking the button again. This behaviour is annoying because the popover moves about if you scroll the page without clicking it off. Its worse when you have multiple popovers open. They move about if they are not clicked closed. When  the trigger set to `focus`, the pop over disappears when you click outside but  does not capture the click event when you click on `Learn more`  i.e. it doesn't navigate away. This allows the popover to stay in place when you click on it so you can click on the link, and it disappears when you click outside it.